### PR TITLE
Splash Screen

### PR DESCRIPTION
**Pull request by esthcarelle**
### Quick Description of What this is about

This PR is all about the splash screen

* I have added the splash api and handled android versions below and above 12 separately(Videos recording shows both of them)
* For the dark and light mode I have added a colors(night mode) file that will be handling colors in dark mode
* The sizes are not yet perfect so I need you to generate another png file that matches the size want
* I also need a brand name png that has a transparent background(The one you gave me has a white background)

https://github.com/cedricbahirwe/dialer-android/assets/52702726/22509773-898b-4992-9997-a1a7bbec5e33

<!-- UI Light                                 |  UI Dark
:-------------------------:|:-------------------------:

### CODE REVIEW

**Comment by cedricbahirwe** [comment]
> ### Quick Description of What this is about
> !! It's not yet ready to be reviewed, I want first to make sure this is what you want PS: WIll upload Dark vs Light mode later [device-2023-10-06-172542.webm](https://github.com/cedricbahirwe/dialer-android/assets/52702726/0d12d4e1-1567-43c1-9210-32fbf415ec95)

That's totally it

**Comment by esthcarelle** [comment]
@cedricbahirwe do you have any idea why I Can't upload files?

**Comment by esthcarelle** [comment]
Second video recording(light Mode and Android 13)
[device-2023-10-09-144205.webm](https://github.com/cedricbahirwe/dialer-android/assets/52702726/2dcd4add-8b68-4cf1-8b78-b3aab9083f40)

**Comment by cedricbahirwe** [comment]
> @cedricbahirwe do you have any idea why I Can't upload files?

Yes You can

**Comment by cedricbahirwe** [comment]
> ### Quick Description of What this is about
> This PR is all about the splash screen
> 
> * I have added the splash api and handled android versions below and above 12 separately(Videos recording shows both of them)
> * For the dark and light mode I have added a colors(night mode) file that will be handling colors in dark mode
> * The sizes are not yet perfect so I need you to generate another png file that matches the size want
> * I also need a brand name png that has a transparent background(The one you gave me has a white background)
> 

The icon I provided, did not have any background color

**Comment by cedricbahirwe** [comment]
> * I have added the splash api and handled android versions below and above 12 separately(Videos recording shows both of them)
> * For the dark and light mode I have added a colors(night mode) file that will be handling colors in dark mode 

Apart from the dark mode background, this looks awesome

**Comment by cedricbahirwe** [comment]
273172129-99057114-0820-4507-b7ec-75f7a8ddc3f9
This one is perfect because it works in both dark and light mode,
What do you think, and it does not have any colored background (dimensions: 200x80)

File: TheDialer/app/src/main/res/drawable/branding_name.png
Comment (cedricbahirwe) [inline TheDialer/app/src/main/res/drawable/branding_name.png:1]: This is the wrong brand icon with the background (My bad if this is what I provided)
Here you go:
273172129-99057114-0820-4507-b7ec-75f7a8ddc3f9

**Review by cedricbahirwe — CHANGES_REQUESTED** [review]

File: README.md
Comment (cedricbahirwe) [inline README.md:1]: Was there a change you needed to make in the README.md file?

Comment (esthcarelle) [reply README.md:1]: @cedricbahirwe Identation was causing red errors with my android studio but you can check there is no difference with what you had

File: TheDialer/app/src/main/res/drawable/branding_name.png
Comment (esthcarelle) [reply TheDialer/app/src/main/res/drawable/branding_name.png:1]: okay thanks

Comment (cedricbahirwe) [reply TheDialer/app/src/main/res/drawable/branding_name.png:1]: Does this work?

Comment (esthcarelle) [reply TheDialer/app/src/main/res/drawable/branding_name.png:1]: yes

Comment (esthcarelle) [reply TheDialer/app/src/main/res/drawable/branding_name.png:1]: yeah

Comment (esthcarelle) [reply TheDialer/app/src/main/res/drawable/branding_name.png:1]: @cedricbahirwe 
https://github.com/cedricbahirwe/dialer-android/assets/52702726/06c6d7f6-9280-4623-a213-21b833b765d4


https://github.com/cedricbahirwe/dialer-android/assets/52702726/d15e4bc9-01f5-435d-9eee-19eb5cc60bd2

Comment (cedricbahirwe) [reply TheDialer/app/src/main/res/drawable/branding_name.png:1]: LGTM, let's launch🚀

**Review by cedricbahirwe — APPROVED** [review]

**Comment by esthcarelle** [comment]
@cedricbahirwe can you first run it on your testing phone to check the drawable sizes(icons)?since we only added one icon it might cause responsiveness issues

**Comment by cedricbahirwe** [comment]
> @cedricbahirwe can you first run it on your testing phone to check the drawable sizes(icons)? since we only added one icon it might cause responsiveness issues

Let me do that

**Comment by cedricbahirwe** [comment]
Looks like the code is not pushed yet

**Comment by cedricbahirwe** [comment]
I'm getting the white background

**Comment by esthcarelle** [comment]
@cedricbahirwe My Bad!You can check now, it was not pushed

**Comment by cedricbahirwe** [comment]
Now, it's working well
https://github.com/cedricbahirwe/dialer-android/assets/49038614/a71dab13-7476-4e61-b236-e06b454074c0

**Pull request merged**